docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+    - Vim.
+    - Sublime Text.
+    - Other (please specify)
+
+6. How do you run and/ or access Jupyter? (pick up to 4)
+    - Run directly on local machine (laptop, desktop).
+    - Through a Python virtual environment (e.g. conda, venv)
+    - Through Docker.
+    - HPC or on-premise server.
+    - Cloud server (e.g. EC2).
+    - JupyterHub.
+    - BinderHub / MyBinder.
+    - Cloud service - AWS: EMR, SageMaker.
+    - Cloud service - Azure: Notebooks, ML Studio.
+    - Cloud service - Databricks
+    - Cloud service - Google: AI Platform, Dataproc.
+    - Cloud service - IBM: Watson.
+    - Google Colab.
+    - Through a 3rd party SaaS app (e.g. Observable, CoCalc)
+    - ❓Don’t know how, I just go to a URL.
+    - Other (please specify)
+
+7. What tasks do you need to perform and what tools do you use to accomplish them?
+    - Writing a software package.
+    - Visualize data in charts, plots, or dashboards.
+    - Writing software documentation.
+    - Building a machine learning or statistical model.
+    - Cleaning and preparing data.
+    - Run pipelines, workflows, or ETL jobs.
+    - Developing extensions/ plugins to solve my problems.
+    - Finding extensions/ plugins to solve my problems.
+    - Creating content (blogs).
+    - Documenting research (reports, scientific papers)
+    - Writing and running tests for software.
+    - Other major use cases (please specify).
+
+    For each of the items above, provide additional information related to:
+    - How frequently do you perform this task?
+        - Never.
+        - Every few months.
+        - Monthly.
+        - Weekly.
+        - Daily.
+    - To what degree does Jupyter meet your expectations for this?
+        - Does not apply.
+        - Never meets.
+        - Rarely meets.
+        - Sometimes meets.
+        - Often exceeds.
+        - Always exceeds.
+    - To what degree do alternative tools meet your expectations for this?
+        - Does not apply.
+        - Never meets.
+        - Rarely meets.
+        - Sometimes meets.
+        - Often exceeds.
+        - Always exceeds.
+
+### Data
+
+8. What data sources are you primarily working with in your role? (pick up to 3)
+    - My local file system (e.g. files and folder on local machine).
+    - File system (e.g. HPC, EBS/EFS, JupyterHub volumes).
+    - Cloud object storage (e.g. buckets, S3, Blob, GS).
+    - SQL (e.g. PostgreSQL, MySQL)
+    - SQL - embedded (e.g. SQLite)
+    - NoSQL - columnar store (e.g. Parquet, Arrow, HDFS, BigQuery).
+    - NoSQL - document store (e.g. MongoDB, Elasticsearch, CouchDB).
+    - Graph database (Neo4j, TigerGraph).
+    - Time Series (e.g. InfluxDB).
+    - Pub sub (e.g. Kafka, Druid).
+    - Key value (e.g. Redis, MemcacheDB).
+    - Google Sheets.
+    - ❗Industry or field specific APIs.
+    - Streaming.
+    - Other (please specify)
+
+9. What data formats are you mostly working with? (pick up to 3)
+    - Tabular (csv, spreadsheet, database tables, parquet).
```

Comment:
Wondering why some questions include `e.g.` when listing examples, but others do not 🤔 ?

Reply:
☑️ changes integrated. Used 'e.g.' everywhere; it reflects syntax/ cody tidiness.